doc/_includes/channel_types.rst

```diff
@@ -23,50 +23,77 @@ parentheses. More information about measurement units is given in the
 .. cssclass:: table-bordered
 .. rst-class:: midvalign
 
-=============  ========================================= =================
-Channel type    Description                              Measurement unit
-=============  ========================================= =================
-eeg            scalp electroencephalography (EEG)        Volts
+================= ========================================= =================
```

Comment:
Can you add a comment suggesting that at some point we should maybe deduplicate this with the glossary? It should be mostly redundant with the glossary entries, right?

Maybe we should use a table in the glossary at some point. Doesn't have to be part of this PR but maybe another comment somewhere that we should...

Reply:
ok I will put for now a reminder comment